src/pages/guide/integration/attributes/data-fixture.md

```diff
@@ -204,6 +204,138 @@ class ProductsListTest extends \PHPUnit\Framework\TestCase
     }
 }
 ```
+### Comment block for data fixture
+
+When writing a data fixture class, it is important to provide a comment block that includes the following information:
+
+-  The usage of the fixture.
+- The example how to use the fixture.
+- For fixtures which can be used in different ways, give example of each use case.
+
+### Examples
+
+Example 1:
+
+```php?start_inline=1
+ /* 
+ *    Target Rule fixture
+ *
+ *    Usage: Using array list. (sku in (simple1,simple3))
```

Comment:
Same here, could you please expand on (sku in (simple1,simple3)), what is this and how does the array below affects it. Again, just to set a very high standard, since this is the goal of this documentation update.

And if you can, please do the same for every example below in this update.